content/articles/2019-10-styler-1.2.0.Rmarkdown

```diff
@@ -0,0 +1,228 @@
+---
+title: "styler 1.2.0"
+author: "Lorenz Walthert"
+date: '2019-10-27'
+description: |
+  Since the initial CRAN submission almost two years ago, many bugs were fixed
+  and new features added, including alignment detection, improved handling of 
+  `{{` and `tryCatch()` expressions and more.
+photo:
+  author: Heng Films
+  url: https://unsplash.com/photos/mpdIPhYqZ4Y
+slug: styler-1.2.0
+categories: ["package"]
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px", 
+  eval = FALSE
+)
+```
+
+The 
+[initial version of styler](https://www.tidyverse.org/articles/2017/12/styler-1.0.0/) 
+was released on CRAN in December 2017. Since then,
+five new versions were released and styler was improved substantially, but many
+improvements were related to special cases or even edge cases. These are all
+contained in the [change log](https://styler.r-lib.org/news/index.html). Here,
+we want to focus on a few changes up to version 1.2.0 that we believe are the 
+most relevant. You can install the latest stable version from CRAN with
+`install.packages("styler")`.
+
+# Alignment detection
+
+styler can finally detect aligned code and keep it aligned! For example, the 
+following code won't be modified by styler:
+
+```{r}
+call(
+  some_arg = 123,
+  more     = "x"
+)
+```
+
+Although quite intuitive, you may need to tweak your aligned code a bit to match
+styler's definition of aligned code. Otherwise, the code will get reformatted
+anyway. Check out the
+[vignette](https://styler.r-lib.org/articles/detect-alignment.html) that
+describes the conditions in both technical and non-technical terms. Note that it
+currently only works for *function calls*, so styler will not treat the below as
```

Comment:
```suggestion
currently only works inside *function calls*, so styler will not treat the below as
```

And I think you can just say "will not modify the following code", and omit the next code block